### PR TITLE
chore: add tests for each query that uses fetch as an http client

### PR DESCRIPTION
**Pull request by soartec-lab**
## Status

**READY**

## Description

follow up #1387 
I add tests for each query that uses fetch as an http client

## Related PRs

none

## Todos

- [x] Tests
- [x] Documentation
- [x] Changelog Entry (unreleased)

## Steps to Test or Reproduce

you can check by execute `yarn test:ci` or seeing if CI is successful.

### CODE REVIEW

**Comment by soartec-lab** [comment]
This PR can be merged after #1518 merged

**Review by melloware — APPROVED** [review]

**Pull request merged**
